server/routerlicious/packages/routerlicious-base/src/alfred/app.ts

```diff
@@ -97,8 +97,9 @@ export function create(
         appTenants,
         documentsCollection);
 
-    app.use("/public", cors(), express.static(path.join(__dirname, "../../public")));
+    app.use("/public", express.static(path.join(__dirname, "../../public")));
```

Comment:
can you remove this entirely?  (and the cors()) .  It is not used any more.

Reply:
I have removed the two lines. 

`app.use("/public", express.static(path.join(__dirname, "../../public")));`
`app.use(cors());`